client/components/plans/plans-compare/index.jsx

```diff
@@ -95,7 +95,8 @@ var PlansCompare = React.createClass( {
 					site={ site }
 					cart={ this.props.cart }
 					features={ featuresList }
-					sitePlans={ this.props.sitePlans } />
+					sitePlans={ this.props.sitePlans }
+					isSubmitting={ this.props.transaction.step.name === 'submitting-wpcom-request' } />
```

Comment:
Yes, or have an helper function in one of the transaction lib such as `isSubmitting( transaction )`